web_responsive/static/src/js/web_responsive.js

```diff
@@ -0,0 +1,238 @@
+/* Copyright 2018 Tecnativa - Jairo Llopis
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+odoo.define('web_responsive', function (require) {
+    'use strict';
+
+    var AppsMenu = require("web.AppsMenu");
+    var config = require("web.config");
+    var core = require("web.core");
+    var Menu = require("web.Menu");
+
+    /**
+     * Recursively search for menus with their full names
+     *
+     * @param {Object} menu
```

Comment:
Can you put a description of what you're expecting, is it a record, or something from another source?